deployments/forgeops/overlay/config/7.0/iot/ds/idrepo/config/schema/thingSchema.ldif

```diff
@@ -3,7 +3,7 @@ changetype: modify
 add: objectClasses
 objectClasses: ( demo-thing
   NAME 'demo-thing'
-  DESC 'Auxiliary class for ForgeRock Things'
+  DESC 'Auxiliary class for ForgeRock IoT'
```

Comment:
I can't decide whether this should be changed or not. Thoughts?

Reply:
Since this is built on top of 7.0 I think we can leave it.